src/static/css/module/brand-header.less

```diff
@@ -0,0 +1,32 @@
+.brand-header {
+  background-color: @gray-10;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  border-bottom: 1px solid @gray-50;
+
+  &__icon {
+    color: @green;
+    font-size: 2.5em;
+    text-align: center;
+    .grid_column(2);
```

Comment:
I'm wondering about the spacing around the icon. On a small screen especially, it seems like it could be left-aligned:

<img width="659" alt="screen shot 2016-11-08 at 10 51 35 pm" src="https://cloud.githubusercontent.com/assets/778171/20129797/43e26160-a607-11e6-8ee0-3fc0f8af7645.png">

Could it be positioned absolutely?